src/Mjolnir.ts

```diff
@@ -282,6 +287,18 @@ export class Mjolnir {
             console.log("Starting web server");
             await this.webapis.start();
 
+            if (this.reportPoll !== undefined) {
+                let reportPollSetting: { from: number } = { from: 0 };
```

Comment:
Since `from` is expected to be opaque, you can't initialize it to `0`. What about making it `{from: number} | null`?

Reply:
> from: integer - Is optional but used for pagination, denoting the offset in the returned results. This should be treated as an opaque value and not explicitly set to anything other than the return value of next_token from a previous call. Defaults to 0.

from https://matrix-org.github.io/synapse/latest/admin_api/event_reports.html

Reply:
My bad :)
In this case, can you add this as documentation?